middle_end/flambda2/to_cmm/to_cmm_env.ml

```diff
@@ -255,13 +255,12 @@ let enter_function_body env ~return_continuation ~exn_continuation =
 
 (* Debuginfo *)
 
-let enter_inlined_apply t dbg =
-  let inlined_debuginfo = Debuginfo.inline t.inlined_debuginfo dbg in
```

Comment:
This is probably the cause of the CI failure: you somehow need to be able to `stack`/`merge` successive `inlined_debuginfos`